extensions/amp-consent/0.1/test/test-consent-ui.js

```diff
@@ -146,7 +146,7 @@ describes.realWin(
         consentUI = new ConsentUI(mockInstance, config);
         expect(consentUI.ui_.tagName).to.equal('IFRAME');
         expect(consentUI.ui_.getAttribute('sandbox')).to.equal(
-          'allow-scripts allow-popups allow-same-origin'
+          'allow-scripts allow-popups allow-popups-to-escape-sandbox allow-same-origin'
```

Comment:
would you mind adding documentation around this

Reply:
we just merged https://github.com/ampproject/amphtml/pull/38250/files, there should be a new "sandbox" property to add these new values. would you mind adding it on to there and then adding docs. 

If not, i can fork this PR to add this request

Reply:
to @JacopoBassan

Reply:
@erwinmombay if you could do that it would be great, I’m on vacation this week so I’m not able to do that, thanks!

Reply:
looks like the other PR i just merged already covers the `allow-popups-to-escape-sandbox` case https://github.com/ampproject/amphtml/pull/38250/files so this should be live in 2 weeks. do verify it though if things look good to you. thank you!

Reply:
@erwinmombay looks good to me thanks, let's wait for the go live

Reply:
@erwinmombay hi, any updates? I'm still not seeing the update in production, have you had any delays? We need to have this asap. Thanks.